lib/index.js

```diff
@@ -447,6 +447,23 @@ exports.withAsyncStorage = (identifier, store, fn) => {
 
 exports.asyncStorageInternals = () => internals.asyncStorageInternals;
 
+exports.patchJoiSchema = (schema) => {
+
+    if (!schema) {
+        return;
+    }
+
```

Comment:
This was part of schwifty's implementation for convenience, but I think here we can assume the user is passing a schema 👍 
```suggestion
```